repos/fdbt-site/src/constants/attributes.ts

```diff
@@ -138,6 +138,8 @@ export const CAPPED_PRODUCT_GROUP_ID_ATTRIBUTE = 'fdbt-capped-product-group-id';
 
 export const CAPS_ATTRIBUTE = 'fdbt-caps';
```

Comment:
Found a couple of bugs when testing it locally:

- Lets users create caps with the same name.
- When editing a cap, the 'Define when cap is calculated from' part shows, even if the duration is less than a day.